actions/matrixsynapse

```diff
@@ -21,11 +21,14 @@
 Configuration helper for Matrix-Synapse server.
 """
 import argparse
+import subprocess
 
 from ruamel.yaml import round_trip_dump, round_trip_load
 
 from plinth import action_utils
 
+MATRIXSYNAPSE_YAML = "/etc/matrix-synapse/homeserver.yaml"
```

Comment:
Minor thing, but would be best defined just once, and import from plinth/modules/matrixsynapse/__init__.py (there, you repeat the raw path anyhow). See solution in current code.